internal/hcs/system.go

```diff
@@ -291,6 +291,10 @@ func (computeSystem *System) waitBackground() {
 	oc.SetSpanStatus(span, err)
 }
 
+func (c *System) WaitChannel() <-chan struct{} {
```

Comment:
Let's use `computeSystem` as the receiver variable here, so we're consistent with the rest of the file.